packages/amplify-provider-awscloudformation/src/utils/override-skeleton-generator.ts

```diff
@@ -0,0 +1,57 @@
+import path from 'path';
+import fs from 'fs-extra';
+import { getPackageManager } from 'amplify-cli-core';
+import execa from 'execa';
+import { $TSContext } from 'amplify-cli-core';
+
+export const generateOverrideSkeleton = async (context: $TSContext): Promise<void> => {
+  let projectPath: string;
+  ({ projectPath } = context.amplify.getEnvInfo());
+
+  // 1. Create skeleton package
+
+  const overrideDirPath = path.join(projectPath, 'amplify', 'backend', 'awscloudformation', 'overrides');
```

Comment:
Should we use PathManager here instead of constructing the path by hand?

Reply:
Yes, will change - thanks for the suggestion.